nx.json

```diff
@@ -98,7 +98,8 @@
       },
       "executor": "@nx/eslint:lint",
       "inputs": ["lint"],
-      "cache": true
+      "cache": true,
+      "dependsOn": ["^build"]
```

Comment:
Not sure this is the correct to requesting the build of the dependencies.
It seems to be a typing issue, the fact to build de depending package may do the job but does not seems to be the correct fix for it.
The typescript path mapping can be the cause of the issue you are facing

Reply:
Thank you for your comment. I had opened this PR to restore the previous working behavior but indeed, it is better to have less dependencies for the lint task and I have opened #1309 to directly address the problem of types.